### DISCUSSION TITLE
Disable Vercel GitHub deploy/bot messages for directories in monorepo

### DISCUSSION THREAD

**Discussion by Southclaws** [vercel/community#182] [Help] [answered]
Hi, we use a monorepo for a project so it contains the frontend (which is deployed via Vercel) and an API, email templates, documentation, and other stuff. When making PRs and changes to the other stuff Vercel pointlessly spends time building and deploying the frontend even though it's identical to the live/master because the changes only affected some random Go code or a readme file.

How do we solve this? How do we make Vercel and the GitHub bot only care about the nodejs app in `frontend/`

Kinda annoying...

![image](https://user-images.githubusercontent.com/1636971/101190525-61bd1f80-3650-11eb-8e96-a7d823b91996.png)

**Comment by paulogdm** [comment, marked as answer]
Check https://vercel.com/knowledge/how-do-i-use-the-ignored-build-step-field-on-vercel. It should have the answer for what you are looking for.

**Reply by Southclaws** [reply]
Thanks!

**Reply by havenchyk** [reply]
@paulogdm sorry if it's not a proper topic, but is there a way to get rid of those messages from vercel bot? I have 

```
"github": {
    "silent": true
  }
```

in vercel.json, but it doesn't help

**Reply by strativd** [reply]
Is there any update on this? It's been an issue for quite some time...

[Here's an issue with more feedback](https://github.com/vercel/vercel/issues/7524). It's especially frustrating as a monorepo/turbo user when every commit has several deploy notifications. PR comments and discussion often get hidden/lost in the "view similar messages" accordion.
